pkg/generated/bindata.go

```diff
@@ -324,6 +478,153 @@ func translationsKubectlEn_usLc_messagesK8sPo() (*asset, error) {
+
+func translationsKubectlFr_frLc_messagesK8sMoBytes() ([]byte, error) {
+	return _translationsKubectlFr_frLc_messagesK8sMo, nil
+}
+
+func translationsKubectlFr_frLc_messagesK8sMo() (*asset, error) {
+	bytes, err := translationsKubectlFr_frLc_messagesK8sMoBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	info := bindataFileInfo{name: "translations/kubectl/fr_FR/LC_MESSAGES/k8s.mo", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
+	a := &asset{bytes: bytes, info: info}
+	return a, nil
+}
+
+var _translationsKubectlFr_frLc_messagesK8sPo = []byte(`# Test translations for unit tests.
+# Copyright (C) 2016
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR brendan.d.burns@gmail.com, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gettext-go-examples-hello\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-12-12 20:03+0000\n"
+"PO-Revision-Date: 2017-01-27 21:04-0800\n"
+"Last-Translator: Brendan Burns <brendan.d.burns@gmail.com>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.6.10\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: fr\n"
+
+msgctxt "Update the annotations on a resource"
+msgid "Update the annotations on a resource"
+msgstr "Mettre à jour les annotations sur une ressource"
+
+msgctxt ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgid ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgid_plural ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgstr[0] ""
+"Watch est uniquement pris en charge sur les ressources individuelles et les "
+"collections de ressources -% d ressource a été trouvée"
+msgstr[1] ""
+"Montre n'est pris en charge que sur les ressources individuelles et les "
+"collections de ressources -% d ressources ont été trouvées"
```

Comment:
Not terribly wrong because understandable but `prendre en charge` has the meaning of taking the responsibility of something, or taking care of something. In an active form, it would have made slightly more sense (`Watch can only take care of ...`), but it is used in passive mode here, which translates literally to something like `Watch is only taken care of *on* (not even by) individual resources...`. Really odd.

I'd use the terms `only compatible with` instead, much simpler construct.

**{0}**
```
watch n'est compatible qu'avec les ressources individuelles et les collections de ressources.  - Aucune ressource n'a été trouvée. 
```

**{1} or {0,1}**
```
watch n'est compatible qu'avec les ressources individuelles et les collections de ressources.  - %d ressource a été trouvée. 
```
(slightly incorrect for 0 though but the way to go if we only have two available forms, as the header suggests - this is typical to encounter that in software but would never happen in life)

**{n > 1}**
```
watch n'est compatible qu'avec les ressources individuelles et les collections de ressources.  - %d ressources ont été trouvées. 
```

Additionally, the fmt tokens were broken. Also, is the format of the file itself correct? Are the empty strings (e.g. msgid, msgctext, ...) normal?